test/crs/test_crs.py

```diff
@@ -1214,13 +1213,14 @@ def test_coordinate_operation_equals():
     assert co != "invalid"
 
 
-@pytest.mark.parametrize(
-    "input_str",
-    ["urn:ogc:def:coordinateOperation:EPSG::1671", "RGF93 v1 to WGS 84 (1)"],
-)
-def test_coordinate_operation__from_string(input_str):
-    co = CoordinateOperation.from_string(input_str)
-    assert co.name == "RGF93 v1 to WGS 84 (1)"
```

Comment:
Mind checking based PROJ version?

https://github.com/pyproj4/pyproj/blob/78b4bed8b42bbba46c46765114b66d84bf1e2d1f/test/crs/test_crs.py#L425

It is a form of documentation & helps to remove the check in the future when the old PROJ version is no longer supported.